backend/cfg/cfg_dataflow.ml

```diff
@@ -405,14 +429,18 @@ module type Backward_S = sig
   val run :
     Cfg.t ->
     ?max_iteration:int ->
+    ?exnescape:domain ->
```

Comment:
It's used in `Zero_alloc_checker.cfg`  for backward analysis on CFG to initialize the blocks that "exit" the function with an exception (instead of a normal return). These are exactly blocks for which `Cfg.can_raise_interproc` returns `true`.
The name `exnescape` is actually from the upstream `Dataflow` analysis. Happy to rename it if you have a suggestion.

Reply:
Sorry, I meant it could be passed another
way, like through the domain, but re-reading
I realize it is kind of the dual to `init` which
is also passed to `run`.